apps/nextjs-website/src/lib/api.ts

```diff
@@ -145,6 +145,37 @@ export async function getVisibleInHomeWebinars(): Promise<readonly Webinar[]> {
   return webinars.filter((webinar) => webinar.isVisibleInHome);
 }
 
+export async function getVisibleInHomeFutureWebinars(): Promise<
+  readonly Webinar[]
+> {
+  return webinars.filter((webinar) => {
+    const startDateTimestamp =
+      webinar?.startDateTime && new Date(webinar.startDateTime).getTime();
+
+    return (
+      webinar.isVisibleInHome &&
+      (!startDateTimestamp || startDateTimestamp > new Date().getTime())
```

Comment:
@MikeAtUqido Is this `new Date().getTime()` executed at build time or at runtime (client-side?)?

Reply:
It's now executed at runtime